### PR TITLE
cleanup feedback suggestions

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Simplified and cleaned up a bunch

# Need Review By (Date)
- 

# Urgency
- 

# Steps to Test

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php [hunk 1]
```diff
@@ -55,17 +54,34 @@ public static function isApplicable(FieldDefinitionInterface $field_definition)
...
+   *   Media type machine name.
+   *
+   * @return \Drupal\media\MediaTypeInterface|null
+   *   Media type entity.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  protected static function getMediaType($media_type_id) {
+    return \Drupal::entityTypeManager()
```
Comment (sherakama) [inline src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: DI

Comment (pookmish) [reply src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: static method. can't use DI

Comment (pookmish) [reply src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: its being used by another static method so thats why.

Comment (sherakama) [reply src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: Oh stupid static.

Comment (sherakama) [reply src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: Protected static tho?

Comment (pookmish) [reply src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php:74]: yeah. it's only used "internally". I've seen it as an approach in Drupal core for some scenarios

**Pull request merged**